### PR TITLE
northstar-css

### PR DESCRIPTION
**Pull request by ssandbekkhaug**
### Description

Please explain the changes you made here.

## Release notes

The title of the pull request will be used as the default release notes description. If more detail is needed to communicate to partners the scope of the PR's changes, use this release notes section.

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [ ] Added the correct label: < feature | enhancement | bug | under-development | dependencies | infrastructure | ignore-for-release | database >
- [ ] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)
- [ ] Added an additional reviewer from outside your organization as FYI (if the primary reviewer is in the same organization as you)
- [ ] Created unit and/or browser tests which fail without the change (if possible)
- [ ] Performed manual testing (Chrome and Firefox if it includes front-end changes)
- [ ] Extended the README / documentation, if necessary

#### Database evolutions

Read the guidelines [here](https://github.com/civiform/civiform/wiki/Database#writing-database-evolutions)

- [ ] Assigned two reviewers
- [ ] Guarded against already existing resources using `IF NOT EXISTS` and `IF EXISTS`
- [ ] Downs created to undo changes in Ups

#### User visible changes

- [ ] Followed steps to [internationalize](https://github.com/civiform/civiform/wiki/Internationalization-%28i18n%29#application-strings) any new strings
  - [ ] Added context strings to new [messages](https://github.com/civiform/civiform/blob/main/server/conf/i18n/messages)
  - [ ] Didn't use a message in applicant facing code that isn't translated yet (unless behind a flag)
- [ ] Wrote browser tests using the [validateAccessibility](https://sourcegraph.com/github.com/civiform/civiform/-/blob/browser-test/src/support/index.ts?L437:14&subtree=true) method
- [ ] Tested on mobile view. See [mobile device mode](https://developer.chrome.com/docs/devtools/device-mode/)
- [ ] Manually tested at 200% size
- [ ] Manually evaluated tab order

#### New Features

- [ ] Add new FeatureFlag env vars to `server/conf/helper/feature-flags.conf`
- [ ] Conditioned new functionality on a [FeatureFlag](https://github.com/civiform/civiform/wiki/Feature-Flags)
- [ ] Wrote browser tests with the feature flag off and on, etc.

### Instructions for manual testing

If instructions are needed for manual testing by reviewers, include them here.

### Issue(s) this completes

Fixes #<issue_number>; Fixes #<issue_number>...

### CODE REVIEW

**Comment by ssandbekkhaug** [comment]
* **#8003** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8003?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ssandbekkhaug and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: server/app/assets/stylesheets/northstar/styles.scss
Comment (gwendolyngoetz) [inline server/app/assets/stylesheets/northstar/styles.scss:1]: Boilerplate

File: server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss
Comment (gwendolyngoetz) [inline server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss:1]: Boilerplate

File: server/app/assets/stylesheets/northstar/_uswds-theme.scss
Comment (gwendolyngoetz) [inline server/app/assets/stylesheets/northstar/_uswds-theme.scss:1]: This is where you can put custom stuff for northstar. There's probably a way to import separate files into this if desired, but we shouldn't need to modify webpack again.

File: server/app/views/BaseHtmlLayout.java [hunk 1]
```diff
@@ -125,8 +125,9 @@ public HtmlBundle getBundle(HtmlBundle bundle) {
 
     // Add default stylesheets.
     bundle.addStylesheets(viewUtils.makeLocalCssTag(USWDS_STYLESHEET_FILEPATH));
+    // TODO: @Don, remove this. Just an example of using it.
+    // bundle.addStylesheets(viewUtils.makeLocalCssTag("dist/uswds_northstar.min"));
```
Comment (gwendolyngoetz) [inline server/app/views/BaseHtmlLayout.java:129]: I didn't feel like constantly flipping the setting. Comment out 127 and uncomment 129 and the main app will use the new northstar stuff now.

You'll want to remove this example, but `dist/uswds_northstar.min` is what you'll want to use when making the link for the css file. It probably needs to be `.min.css`, I'm guessing the viewutils add that.

File: server/app/views/NorthStarBaseView.java [hunk 2]
```diff
@@ -58,6 +58,7 @@ protected ThymeleafModule.PlayThymeleafContext createThymeleafContext(
     ThymeleafModule.PlayThymeleafContext context = playThymeleafContextFactory.create(request);
     context.setVariable("tailwindStylesheet", assetsFinder.path("stylesheets/tailwind.css"));
     context.setVariable("uswdsStylesheet", assetsFinder.path("dist/uswds.min.css"));
+    context.setVariable("northStarStylesheet", assetsFinder.path("dist/north-star-styles.css"));
```
Comment (gwendolyngoetz) [inline server/app/views/NorthStarBaseView.java:61]: As mentioned above, use `dist/northstar-styles.min.css`. I renamed it because I kept typing it wrong

**Pull request closed without merging**